roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -30,13 +30,19 @@ def get_info_from_hammer(command, column=1)
     external_capsules << {:id => id, :name => name, :lifecycle_environments => lifecycle_environment}
   end
 
-
   reverse_commands = []
+  satellite_version = ARGV[0].to_f
   external_capsules.each do |capsule|
     capsule[:lifecycle_environments].each do |env|
-      run_hammer_cmd("--csv capsule content remove-lifecycle-environment --id #{capsule[:id]} --environment-id #{env}")
-      reverse_command = prepare_hammer_cmd("--csv capsule content add-lifecycle-environment --id #{capsule[:id]} --environment-id #{env}")
-      reverse_commands << reverse_command
+      if satellite_version > 6.5
```

Comment:
It shouldn't make a difference with two digits, but it is nice to use `Gem::Version` to compare versions to make sure the version math is done correctly. You'll need to add `require "rubygems"` at the top as well
```suggestion
      if Gem::Version.new(satellite_version) > Gem::Version.new(6.5)
```

Reply:
Fixed